src/framework/Elsa.Studio.Core/Elsa.Studio.Core.csproj

```diff
@@ -33,12 +33,12 @@
         <PackageReference Include="ThrottleDebounce"/>
     </ItemGroup>
 
-    <!--    <ItemGroup Label="Elsa">-->
-    <!--        <ProjectReference Include="..\..\..\..\..\elsa-core\3.2.x\src\clients\Elsa.Api.Client\Elsa.Api.Client.csproj"/>-->
-    <!--    </ItemGroup>-->
-
     <ItemGroup Label="Elsa">
-        <PackageReference Include="Elsa.Api.Client"/>
+        <ProjectReference Include="..\..\..\..\..\elsa-core\3.2.x\src\clients\Elsa.Api.Client\Elsa.Api.Client.csproj"/>
     </ItemGroup>
 
+    <!--    <ItemGroup Label="Elsa">-->
+    <!--        <PackageReference Include="Elsa.Api.Client"/>-->
+    <!--    </ItemGroup>-->
+
```

Comment:
Will flip this back as soon as the [elsa-core PR](https://github.com/elsa-workflows/elsa-core/pull/6018) has been merged and new packages are available.